iml-wire-types/wire-types.rs

```diff
@@ -542,7 +542,7 @@ pub struct Volume {
     pub kind: String,
     pub label: String,
     pub resource_uri: String,
-    pub size: Option<String>,
+    pub size: Option<i64>,
```

Comment:
Is there ever a case in which size could be negative?

Reply:
I hope not, it would not be a very useful storage device :)

Reply:
Should it be a u64?

Reply:
Ah, right. I remember why I did this. The volume size is stored in the database as a `BIGINT` which has a max value of `9223372036854775807` Which corresponds to the max_value of an i64 in rust: https://doc.rust-lang.org/std/primitive.i64.html#method.max_value. If we were to use a u64 instead, it would have the potential to overflow, as it's max value is `18446744073709551615`

I'm in the middle of redoing device persistence in IML, I guess I should use https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.DecimalField for the update.

Reply:
Ok that makes sense 👍